Formula/qwt.rb

```diff
@@ -29,12 +29,18 @@ def install
     end
 
     args = ["-config", "release", "-spec"]
-    spec = if ENV.compiler == :clang
-      "macx-clang"
-    else
-      "macx-g++"
+    spec = nil
+    on_macos do
+      spec = if ENV.compiler == :clang
+        "macx-clang"
+      else
+        "macx-g++"
+      end
+      spec << "-arm64" if Hardware::CPU.arm?
+    end
+    on_linux do
+      spec = "linux-g++"
     end
```

Comment:
You can make things a bit easier to read (I think) and minimise the number lines you need to touch by just adding this `on_linux` block after all the macOS stuff with `spec` is done.

Reply:
...or since you have another pair of `on_macos` and `on_linux` blocks below, just put all the macOS stuff together and all the Linux stuff together. No need to jump back and forth between them.

Reply:
Done